Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,23 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+
+const CardContent = () => {
+  return (
+    <div className="cardContent">
+      <a href="https://reactjs.org/">
+        <section class="contentSection">
+          <h2>Get started with React</h2>
+          <p>
+            React makes it painless to create interactive UIs. Design simple
+            views for each state in your application.
+          </p>
+        </section>
+        <a className="cardAnchor" href="https://reactjs.org/">
```

Comment:
since you have 'readctjs.org' as an href on a higher a tag, it will go there anyway, so your href for this anchor tag can be 'href='#'' this might fix some bugs and stop repetition, but still give it the styles that an anchor tag has.